src/ui/RealmBrowser/index.tsx

```diff
@@ -399,6 +403,26 @@ class RealmBrowserContainer
     this.performImport();
   };
 
+  private isEmbeddedType = (className: string): boolean => {
+    const { classes } = this.state;
+    return classes.find(c => c.name === className)?.embedded ?? false;
+  };
+
+  private isCreateAllowed = (focus?: Focus): boolean => {
+    if (focus && focus.kind === 'class') {
+      return !this.isEmbeddedType(focus.className);
+    } else if (focus && focus.kind === 'list') {
+      // TODO: Warning, not entirely sure about this one!
+      return !focus.property.isEmbedded;
+    }
+
+    return false;
+  };
+
+  private getFirstSchemaName = () =>
+    this.realm?.schema.find(c => c.name.indexOf('__') !== 0 && !c.embedded)
+      ?.name;
```

Comment:
Should we move this to ui/RealmBrowser/schema-utils.ts?

Reply:
It's not used anywhere else (anymore), so I'll move the check back in-line.